tests/test_utils.py

```diff
@@ -189,14 +198,21 @@ def get_dataset(
                 elif type_ in ["DATETIME", "TIMESTAMP"]:
                     df[col] = pd.to_datetime(df[col])
 
-        tablename = "test_data_" + "".join(
-            [random.choice(string.ascii_letters + string.digits) for n in range(8)]
+        if table_name is None:
+            table_name = "test_data_" + "".join(
+                [random.choice(string.ascii_letters + string.digits) for _ in range(8)]
+            )
+        df.to_sql(
```

Comment:
What's the advantage of using SQLite as a temporary store to run tests? Is it faster? Less prone to schema errors?

Reply:
I'm not sure what you're comparing *to* in that question...
If before/after:
Before, each test was run in an isolated sqlite in-memory database. Since this new expectation requires a database with *multiple* tables in it, it became necessary to persist those databases in some cases.
If vs. postgres or some other db:
sqlite is distributed as part of python, so it's available for unit tests independent of other integrations.

Reply:
Oops. This was a misread on my part. I get it now. Carry on. :)